src/components/index.ts

```diff
@@ -0,0 +1,5 @@
+export { default as RadarAnimation } from './radar';
+export * from './radar';
```

Comment:
You not only export the component default as named, but also all types exports. This will be useful when generating .d.ts files at build time, the root index.d.ts will be pointed to within package.json